lib/Sema/DerivedConformanceParameterAggregate.cpp

```diff
@@ -232,7 +231,7 @@ static ValueDecl *deriveParameterAggregate_update(DerivedConformance &derived) {
   }
   updateDecl->setInterfaceType(interfaceType);
```

Comment:
Instead, please just call `updateDecl->computeType()`, and remove the code for building your own `FunctionType`.

Reply:
@dan-zheng Please take a look. Thanks!

Reply:
@dan-zheng please take a look. thanks!